sfge/src/test/java/com/salesforce/graph/ops/expander/ApexPathExpanderInLoopTest.java

```diff
@@ -4,18 +4,50 @@
 
 import com.salesforce.TestRunner;
 import com.salesforce.TestUtil;
+import com.salesforce.graph.symbols.SymbolProvider;
+import com.salesforce.graph.symbols.apex.ApexBooleanValue;
 import com.salesforce.graph.symbols.apex.ApexForLoopValue;
 import com.salesforce.graph.symbols.apex.ApexStringValue;
+import com.salesforce.graph.symbols.apex.ApexValue;
+import com.salesforce.graph.vertex.ChainedVertex;
+import com.salesforce.graph.vertex.MethodCallExpressionVertex;
 import com.salesforce.graph.visitor.SystemDebugAccumulator;
 import java.util.List;
 import java.util.stream.Collectors;
 import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+/**
```

Comment:
Javadoc explains the differences between method calls on various types of array items.